terraform/modules/bastion/iam.tf

```diff
@@ -23,4 +23,23 @@ resource "aws_iam_role_policy_attachment" "ssm_bastion_attachment" {
 resource "aws_iam_instance_profile" "ssm_bastion" {
   name = "${var.environment_name}-bastion-instance-profile"
   role = aws_iam_role.ssm_bastion.name
-}
\ No newline at end of file
+}
+
+resource "aws_iam_role_policy_attachment" "ssm_bastion_maintenance_window" {
```

Comment:
I might have another check in the documentation for whether we need to change these roles if if it will do it by default